pkg/apis/stork/v1alpha1/migrationschedule.go

```diff
@@ -15,6 +15,7 @@ const (
 type MigrationScheduleSpec struct {
 	Template           MigrationTemplateSpec `json:"template"`
 	SchedulePolicyName string                `json:"schedulePolicyName"`
+	Enabled            *bool                 `json:"enabled"`
```

Comment:
Wouldn't calling this as "Disabled" make it simple if the default is to keep it enabled?
We won't have to set the defaults.

Reply:
Going to rename this to `suspend` to match CronJobs: https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#suspend

Reply:
Still leaving the code to set the defaults so that it shows up when users check the spec.

Reply:
Nevermind, removed it